docs/2.7.0/docs/ops/requirements.rst

```diff
@@ -8,7 +8,9 @@ System Requirements
 
 Unless otherwise stated, all Daml runtime components require the following dependencies:
 
-1. An x86-compatible system running a modern Unix, Windows, or MacOS operating system.
+1. For development, an x86-compatible system running a modern Linux, Windows,
+   or MacOS operating system.  For a production deployment, an x86-compatible
```

Comment:
Minor nitpick: it may be worth mentioning that the SDK does run under Rosetta2 (i.e. runs on m1 macs).

Reply:
I think this is implied since we don't single out Intel based Macs.

Reply:
Yes, it is implied. I was saying I think it would be better if it were explicit, as it's easy to miss right now. It's easy to read x86-compatible mac as Intel mac.